x-pack/libs/es-opensaml-security-api/src/main/java/org/opensaml/security/config/org/elasticsearch/xpack/security/authc/saml/OpenSamlXpackSecurityConfigurationPropertiesSource.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.opensaml.security.config.org.elasticsearch.xpack.security.authc.saml;
```

Comment:
Note the strange package name.
A package can belong to a single module and the shadowed jar defines the automatic module `org.opensaml.security`. I cannot use the `org.elasticsearch.xpack.security.authc.saml` package because that's exported by the `oeg.elasticsearch.security` module.

Reply:
I'd probably prefer something shorter like
```suggestion
package org.opensaml.security.config.org.elasticsearch;
```
but I'm not fussed.

Reply:
Pushed [d6e652c](https://github.com/elastic/elasticsearch/pull/98199/commits/d6e652cee34c2c26d6fdc0b49b0ec167cdb852f3)